### ISSUE TITLE
Windows 10 Triton Client Build Does Not Generate a .whl file

### ISSUE THREAD

**Issue by JulianOrteil** [NVIDIA/triton-inference-server#1423]
**Description**
Hello, I am attempting to compile Triton Clients on Windows 10 for Python 3.7.  I followed your walkthrough [here](https://github.com/NVIDIA/triton-inference-server/blob/master/docs/client.rst#windows-10) but have been unable to get the wheel files to generate. While the build process completes without errors, there are a multitude of warnings produced (most of them being duplicates).

The example files build and generate without issue, it is only the whl file that I can't generate.

Here are 2 warnings that are duplicated multiple times which I found to be somewhat relevant:
![image](https://user-images.githubusercontent.com/23083317/81335063-a1953780-906c-11ea-9a0d-eb0b9b3d1dd1.png)

Please note that I had to rename the root folder from "tensorrt-inference-server" to "trtis" as msbuild kept throwing a "name is too long" error.

I tried running this as both a normal user and as administrator with no success.

**Triton Information**
I am checking out r20.03 when building.

Are you using the Triton container or did you build it yourself?
Built myself (client side only)

**To Reproduce**
Steps to reproduce the behavior.
Follow the guide above.

Describe the models (framework, inputs, outputs), ideally include the model configuration file (if using an ensemble include the model configuration file for that as well).
N/A

**Expected behavior**
The whl file to be generated.

**Comment by dzier** [comment]
Thank you for finding this issue. It was reproducible on our end. We are working on a solution to the issue.

**Comment by JulianOrteil** [comment]
I noticed a pull request from you @dzier regarding this. If I build the clients with the current master branch (as this is a little time sensitive), will I be able to communicate with the server itself?

And thank you for a timely fix.

**Comment by dzier** [comment]
Yes, I was waiting until our internal CI passed. The fix is complete and has been pushed into master. It will also be added to the next 20.05 release. It should work with 20.03 as long as the server is still using the v1 API. To be on the safe side, you can make the same change to your local copy of the 20.03 repo and rebuild the windows client. It is just a single line change.

Closing the issue as fixed, please re-open or file a new issue if the problem persists.

**Comment by JulianOrteil** [comment]
Still haven't been able to get a wheel file to generate. I looked at the path you entered and do not see the "Release" folder in src\clients\c++\api_v1\library that you reference. I see the "request" C++ source file, but not a Release folder or the compiled request.dll file.

![image](https://user-images.githubusercontent.com/23083317/81577598-3c448d80-936f-11ea-8e57-d5ec620c11f6.png)

I looked through the rest of the log outputted to the command prompt and see "successful build" for the other required dependencies (curl, grpc, etc).

**Comment by JulianOrteil** [comment]
However after further digging, I do see the request.dll file here:
![image](https://user-images.githubusercontent.com/23083317/81578449-719dab00-9370-11ea-9018-6cf5e0402591.png)

**Comment by dzier** [comment]
All build collateral will be in the build directory. After you build, you should see the following directory:

```
trts\build\trtis-clients\install\python
```

That will contain the *pb2 files and the whl file.

**Comment by JulianOrteil** [comment]
This is all I got:
![image](https://user-images.githubusercontent.com/23083317/81584856-ed9bf100-9378-11ea-8f04-26054a69d887.png)

**Comment by dzier** [comment]
You will also need to make sure that you have at least python 3.5 installed, referenced as `python3.exe` and have it available in the windows path. There are also a few libraries needed from `pip`. So try running the following command.

```
pip3 install --upgrade setuptools wheel grpcio
```

**Comment by dzier** [comment]
I found in my test, that the wheel file was failing to build in our CI because python was not set up correctly.

To build manually and see the output, if it is python, you can run the following in a bash window in the `build\trtis-clients\python\api_v1\library`

```
VERSION=0.0.0 python3 setup.py bdist_wheel
```

This will get ran in the normal build, but runs in a separate bash window that often exists before you can see the error.

**Comment by JulianOrteil** [comment]
Okay, I figured out what my issue was.

Visual Studio Code.

I started uninstalling programs that might have been causing issues and determined that VSCode was the issue. I was able to generate the wheel file using the standard build procedure in the Wiki. Would there be a reason why VSCode would be the issue? Maybe it was selected as the default file opener for the file generating the wheel file?
